include/core/truthTable/truth_table.hpp

```diff
@@ -313,6 +359,33 @@ namespace syrec {
             return cubeMap.find(c);
         }
 
+        template<class It>
+        auto erase(It elem) -> It {
+            return cubeMap.erase(elem);
+        }
+
+        auto resizeConstants(const std::size_t n) -> void {
+            constants.resize(n);
+        }
+
+        auto resizeGarbage(const std::size_t n) -> void {
+            garbage.resize(n);
+        }
+
+        auto resetConstants() -> void {
+            constants.clear();
+        }
+
+        auto resetGarbage() -> void {
+            garbage.clear();
+        }
```

Comment:
These wrapper should not be necessary as pointed out by one of the comments below.

Reply:
Sounds reasonable.